src/browser/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+if(WITH_XC_BROWSER)
+    include_directories(${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR})
+    
+    find_package(Boost 1.55 COMPONENTS system REQUIRED)
+    include_directories(${Boost_INCLUDE_DIR})
+    link_directories(${Boost_LIBRARY_DIRS})
+    set(Boost_USE_STATIC_LIBS           OFF)
```

Comment:
At first I tried purely Qt based solution but it didn't work as expected. Using QSocketNotifier with stdin is going to put the CPU% to the maximum. I didn't manage to found a workaround for that issue.